### ISSUE TITLE
[TypeScript] Error when using tsconfigPath with buildMode: true

### ISSUE THREAD

**Issue by shayded-exe** [fi3ework/vite-plugin-checker#181]
### Describe the bug

When specifying both a `tsconfigPath` and `buildMode: true`, the TS checker specifies both the `-b` and `-p` flags to `tsc` which is an error.

### Reproduction

Specify both `tsconfigPath` and `buildMode: true`

### Expected behavior

tsc runs without error

### System Info

```shell
System:
    OS: Windows 10 10.0.19044
    CPU: (16) x64 AMD Ryzen 7 5800X 8-Core Processor
    Memory: 19.77 GB / 31.93 GB
  Binaries:
    Node: 16.14.1 - C:\Program Files\nodejs\node.EXE
    npm: 8.19.2 - C:\Program Files\nodejs\npm.CMD
  Browsers:
    Chrome: 106.0.5249.119
    Edge: Spartan (44.19041.1266.0), Chromium (106.0.1370.52)
    Internet Explorer: 11.0.19041.1566
```
```


### Additional context

_No response_

### Validations

- [X] Read the [docs](https://github.com/fi3ework/vite-plugin-checker#readme).
- [X] Check that there isn't already an issue that reports the same bug to avoid creating a duplicate.

**Comment by fi3ework** [comment]
fixed in #186

**Comment by shayded-exe** [comment]
Awesome!

The issue stated in #186 wasn't even a problem I knew about, so it's cool that it got addressed.

In my case I'm not emitting declaration files, I need to use build mode because I have composite projects. I actually ended up not using the `tsconfigPath` setting and instead set `root` to my src folder, which has been working well for me.
